streaming/rdm_atomic.c

```diff
@@ -274,6 +273,16 @@ static int run_op(void)
 			}
 		}
 		break;
+	case FI_ATOMIC_READ:
+		ret = is_valid_fetch_atomic_op(op_type);
+		if (ret > 0) {
+			for (i = 0; i < opts.iterations; i++) {
+				ret = execute_fetch_atomic_op(op_type);
+				if (ret)
+					break;
+			}
+		}
+		break;
 	case FI_CSWAP:
```

Comment:
This looks fine. But I think we can avoid code duplication by just moving 'case FI_ATOMIC_READ'.